api/app/GraphQL/Validators/Mutation/ChangeApplicationSuspendedAtValidator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\GraphQL\Validators\Mutation;
+use Database\Helpers\ApiEnums;
+use Illuminate\Validation\Rule;
+use Nuwave\Lighthouse\Validation\Validator;
+
+final class ChangeApplicationSuspendedAtValidator extends Validator
+{
+    /**
+     * Return the validation rules.
+     *
+     * @return array<string, array<mixed>>
+     */
+    public function rules(): array
```

Comment:
Feel free to disagree with this one.  :smile: I'm uncomfortable that you can "suspend" already suspended records and update the suspended timestamp.  I'd like to see a validation error but I'd be happy with silently discarding the request as well.  Or maybe it's not a big deal?

Reply:
I updated to go with the silent discarding in [bd4c41d](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5832/commits/bd4c41d0f15733587082cf6e8d8acf9e481b12a7).